src/store/index.js

```diff
@@ -5,8 +5,12 @@ Vue.use(Vuex)
 
 export default new Vuex.Store({
   state: {
+    userName: ''
   },
```

Comment:
userに関するstateはこの後も増えていくかと思います。
その都度user〇〇と命名してくのは冗長なので、、一度userという命名でオブジェクトを作成し、そこに情報を格納していく方法が管理しやすいかと思います。

```suggestion
  state: {
    user {
      name: '',
    }
  },
```